model-optimizer/extensions/front/reduce_axis_normalizer.py

```diff
@@ -69,3 +70,10 @@ def replace_sub_graph(self, graph: Graph, match: [dict, SubgraphMatch]):
                 node.add_input_port(1, skip_if_exist=True)
                 axes.out_port(0).connect(node.in_port(1))
                 node.in_port(0).get_connection().get_source().connect(end_of_range.in_port(0))
+
+            # We need to add normalization order value for ReduceLp operation as 3rd input
+            if node.has('p'):
```

Comment:
This change is not related to the axis normalization. So I would recommend to move it to a separate transformation. Moreover, it is applicable for ReduceLp only while this one applies for all Reduce ops.

Reply:
Separate transformation added.